Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -276,50 +271,21 @@ public Command getAutonomousCommand( )
     {
       default :
       case AUTOSTOP :
-      case AUTOPRELOADONLY :
         break;
       case AUTOLEAVE :
-      case AUTOPRELOADANDLEAVE :
-        pathName = (alliance == Alliance.Red) ? "leaveStartingZoneRed" : "leaveStartingZoneBlue";
-        break;
-      case AUTOPRELOADSCOREANOTHER :
-        pathName = (alliance == Alliance.Red) ? "driveToAnotherRed" : "driveToAnotherBlue";
+        pathName = (alliance == Alliance.Red) ? "R_Note1Acquire" : "B_Note1Acquire";
```

Comment:
You're probably still working on this, but we shouldn't need both red and blue paths, right? And the path names should match the enum names so we can keep them straight.

Reply:
Also can be merged and fixed later for this one.